### PR TITLE
Hotel Security spellchecked

### PR DESCRIPTION
**Pull request by Mercenaryblue**
[Changelogs]
:cl: 
spellcheck: spellchecked the hotel staff.
/:cl:

_interests_

### CODE REVIEW

**Comment by YakumoChen** [comment]
we still have the hotel??

**Comment by Mercenaryblue** [comment]
Its in the files.

**Review by Cyberboss — APPROVED** [review]

**Comment by PKPenguin321** [comment]
If you need instructions, on how to get through the hotel, check out the enclosed instruction book

**Comment by ShizCalev** [comment]
Terrible changelog.

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -319,7 +319,7 @@
 	name = "hotel security sleeper"
 	mob_name = "hotel security memeber"
```
Comment (ShizCalev) [inline code/game/objects/structures/ghost_role_spawners.dm]: fix `memeber` too please.

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 2]
```diff
@@ -319,7 +319,7 @@
...
 	outfit = /datum/outfit/hotelstaff/security
-	flavour_text = "You are a peacekeeper assigned to this hotel to protect the intrests of the company while keeping the peace between \
+	flavour_text = "You are a peacekeeper assigned to this hotel to protect the interests of the company while keeping the peace between \
```
Comment (ShizCalev) [inline code/game/objects/structures/ghost_role_spawners.dm]: Space after the period following staff.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 3]
```diff
@@ -319,7 +319,7 @@
...
 		guests and the staff.Do <font size=6><b>NOT</b></font> leave the hotel, as that is grounds for contract termination."
```
Comment (ShizCalev) [inline code/game/objects/structures/ghost_role_spawners.dm]: Space after the period following staff.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request merged**